docs: fix linter errors for lazy-loading-ngmodules.md (#47645)
angular/angular: aio/content/guide/lazy-loading-ngmodules.md

A common error when lazy-loading modules is importing common modules in multiple places within an application.
Test for this condition by first generating the module using the Angular CLI and including the `--route route-name` parameter, where `route-name` is the name of your module.
Next, create the module without the `--route` parameter.

> If the Angular CLI generates an error when using the `--route` parameter, but runs correctly without it, you might have imported the same module in multiple places.

Reviewer: Try this:

```suggestion
If `ng generate module` with the `--route` parameter returns an error, but runs correctly without it, you might have imported the same module in multiple places.
```

Anywhere you can replace `Angular CLI` with an actual command, that generally makes the sentence clearer and easier to understand. Generally, such cases are when `Angular CLI` is attributed with performing some action (creating a module, in this case). If you'd like to review the topic to make similar changes, that'd be great. If not, that's fine, too.

As an anecdote, I spent about 5 minutes looking up the replacement command. I found `ng generate module`, which I think would the replacement in this example. Had the original writer used that instead of `CLI` or `Angular CLI`, I'd have saved those 5 minutes. If just 1% of the readers of that page had the same experience, fixing just this one instance would save its readers 38 hours a month.

This isn't called out by the linter (yet), because our initial tests of that rule had too many false positives. This is mostly due to how generic `CLI` is and we didn't have much luck trying to tune the rule to figure out whether `CLI` was used in an appropriate context or not.

Author: Hi @bob-watson,

Thanks for the explanation about using Angular CLI in documentation, I'll keep it in mind for future changes!
I clicked the "Commit Suggestion" below your post, but I'm not sure how to squash both commits now.

Is this something you can do on your end?

Reviewer: [Update a documentation pull request](https://pr47381-0644028.ngbuilds.io/guide/doc-pr-update#clean-up-the-branch) is a documentation update that's in review and talks about squashing your commits.
Be sure that the resulting commit message has the correct format.
